README.md

```diff
@@ -4,7 +4,9 @@ This is a plugin for a local DynamoDB instance using Docker.
 
 When you are using Architect's sandbox mode, the plugin starts a Docker container running the [`amazon/dynamodb-local:latest`](https://hub.docker.com/r/amazon/dynamodb-local) image.
 
-This can be used alongside other plugins such as [@hicksy/arc-plugin-sandbox-stream](https://github.com/hicksy/arc-plugin-sandbox-stream) to allow for more realistic debugging with `tables-streams`.
+If you have defined a `@tables-streams` section in your `app.arc` file, the respective streams will be enabled.
```

Comment:
I think what you mean is that the plugin will automatically invoke the configured Lambdas.

Reply:
Well yea, these are both true statements. Streams will be enabled and they will be triggered

Reply:
I'm just thinking that "the respective streams will be enabled" is a bit cryptic.

Reply:
Ya, I will reword it in the next commit